{"classFile":"ExceptionUtilsTest","className":"ExceptionUtilsTest","packages":"org.apache.commons.lang3.exception","methodInfos":[{"md5":"IK5Mv9F1wh0DWUIvvIUBsw==","methodName":"testGetRootCauseStackTrace_Throwable","parameters":"[]"}],"addLines":[411,412,421,422],"delLines":[411,412,421,422],"type":"MODIFY","path":"src/test/java/org/apache/commons/lang3/exception/ExceptionUtilsTest.java","pro_Commit":"fab64bbdc726cf06c5993b7b8f50557882086002","pro_Date":"2011-07-11 02:26:16","sha1":"370ff58dd0303fe6932cc1c3bbd31263db0e8b54","file_Date":"2011-07-11 02:26:16","projectName":"commons-lang","commitID":"fab64bbdc726cf06c5993b7b8f50557882086002"}